practice/pyramid/site.css

```diff
@@ -0,0 +1,22 @@
+.row {
+  box-sizing: border-box;
+  height: 100px;
+  padding: 20px;
+  display: flex;
+  justify-content: space-around;
+}
+
+.inner {
```

Comment:
Instead of creating a new class for every inner div, you could select on `.row > div` and automatically get that. If you're applying a class to every element in a certain position, use a structure selector instead.